usr/src/cmd/zfs/zfs_main.c

```diff
@@ -5395,8 +5395,13 @@ print_holds(boolean_t scripted, size_t nwidth, size_t tagwidth, nvlist_t *nvl)
 			(void) strftime(tsbuf, DATETIME_BUF_LEN,
 			    gettext(STRFTIME_FMT_STR), &t);
 
-			(void) printf("%-*s%*c%-*s%*c%s\n", nwidth, zname,
-			    sepnum, sep, tagwidth, tagname, sepnum, sep, tsbuf);
+			if (scripted)
+				(void) printf("%s%c%s%c%s\n", zname,
+				    sep, tagname, sep, tsbuf);
+			else
+				(void) printf("%-*s%*c%-*s%*c%s\n", nwidth,
+				    zname, sepnum, sep, tagwidth, tagname,
+				    sepnum, sep, tsbuf);
```

Comment:
The printf's would be simpler if we used ` ` / `\t` in the format string, rather than using the variable `sep`.

For readability, cstyle recommends {braces} around multi-line if/else bodies (even if only one statement).